dto.c

```diff
@@ -23,6 +24,7 @@
 #include <accel-config/libaccel_config.h>
 #include <numaif.h>
 #include <numa.h>
+#include <linux/limits.h>
```

Comment:
The code compiles without this #include. Can you please check if it is really needed?